.lintstagedrc

```diff
@@ -0,0 +1,11 @@
+{
+    "packages/*/package.json": [
```

Comment:
There is no `packages` directory. This could just run on `package.json`, I guess?

Reply:
good catch 👍